gromacs/testing.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2009 Oliver Beckstein <orbeckst@gmail.com>
+# Released under the GNU Public License 3 (or higher, your choice)
+# See the file COPYING for details.
+
+import os, inspect
+
+def get_data_path(f=''):
+    """Return the path of a data file, these are relative to the current test
+    directory.
+    """
+    # get our callers file
+    _, filename, _, _, _, _ = inspect.getouterframes(inspect.currentframe())[1]
```

Comment:
@orbeckst This does look kind of ghetto but is line by line from pandas testing. I'm going to assume they know what they're doing. 

To understand that consider the following ipython example
```
In [2]: import inspect 

In [3]: inspect.currentframe()
Out[3]: <frame at 0x1dbbf30>

In [4]: inspect.getouterframes(inspect.currentframe())[1]
Out[4]: 
(<frame at 0x7f89bed6a850>,
 '/usr/lib/python2.7/dist-packages/IPython/core/interactiveshell.py',
 2820,
 'run_code',
 ['                exec code_obj in self.user_global_ns, self.user_ns\n'],
 0)

```

So basically it's all good and kosher here, with a bit of introspection.